cpp/tests/framework/Engine.test.cpp

```
@@ -7,14 +7,22 @@
+#include <algorithm>
 #include <arpa/inet.h>
 #include <atomic>
 #include <catch2/catch_all.hpp>
 #include <chrono>
 #include <netinet/in.h>
+#include <nlohmann/json.hpp>
 #include <sys/socket.h>
 
+namespace robotick
+{
+	nlohmann::ordered_json build_workloads_buffer_layout_json(const Engine& engine, const char* session_id_override);
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)